cpp/lib/ome/bioformats/detail/FormatWriter.h

```diff
@@ -121,6 +121,9 @@ namespace ome
         /// Current series.
         mutable dimension_size_type series;
 
+        /// Current plane.
+        mutable dimension_size_type plane;
```

Comment:
`dimension_size_type` makes sense for plane number too? (I don't object, it's just not what I'd have guessed.)